### DISCUSSION TITLE
How to hide 'create new' when there is no input but focused

### DISCUSSION THREAD

**Discussion by neostom432** [JedWatson/react-select#5289] [Q&A]
Hi, I'm using `Creatable` in `react-select/creatable `and want to hide 'Create new' when user doesn't type anything.
For now, when user click selector, 'Create new' option is showing even user doesn't type.

How can I show 'Create new' when only user types something?

**Comment by manjushsh** [comment]
> For now, when user click selector, 'Create new' option is showing even user doesn't type.

How to reproduce this?

**Reply by neostom432** [reply]
Hello @manjushsh,
It seems that It only happened my own custom select.

https://gist.github.com/neostom432/8c349257b1f7497f71273b9ce19817f9

Could you check my component?
